yarn.lock

```diff
@@ -2527,7 +2527,7 @@ eslint-plugin-flowtype@^2.30.0:
     lodash "^4.15.0"
 
 "eslint-plugin-guardian-frontend@file:dev/eslint-plugin-guardian-frontend":
-  version "3.0.0"
+  version "4.0.0"
```

Comment:
missed this, sorry. yeah lock file needs to be updated when we bump that version in package. you can do it manually or remove/re-add it to have yarn do it